src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -156,6 +223,137 @@ class PostmanSmsService {
       )
     })
   }
+
+  public sendBouncedSubmissionSms(
+    recipient: string,
+    recipientEmail: string,
+    adminId: string,
+    adminEmail: string,
+    formId: string,
+    formTitle: string,
+  ): ResultAsync<true, SmsSendError | InvalidNumberError> {
+    const logMeta = {
+      action: 'sendBouncedSubmissionSms',
+      formId,
+    }
+
+    logger.info({
+      message: `Sending bounced submission notification for ${recipientEmail}`,
+      meta: logMeta,
+    })
+
+    if (!isPhoneNumber(recipient)) {
```

Comment:
Can move this check to `_sendInternalSms` to avoid duplication

Reply:
Should be resolved at the dedup-ing of caller!